From 8f9a4eba419ffb193f63fac15dba1f814413e4e6 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Thu, 25 Jul 2024 22:24:51 +0700
Subject: [PATCH] rz-common: recipes-debian: buster: glibc: update to
 v2.28-10+deb10u4

Update glibc to v2.28-10+deb10u4 based on its changelog:
* Non-maintainer upload by the LTS Team.
* CVE-2024-33599: nscd: Stack-based buffer overflow in netgroup cache
* CVE-2024-33600: nscd: Null pointer crashes after notfound response
* CVE-2024-33601: nscd: Daemon may terminate on memory allocation failure
* CVE-2024-33602: nscd: Possible memory corruption

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 .../recipes-debian/buster/sources/glibc.inc        | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/meta-rz-common/recipes-debian/buster/sources/glibc.inc b/meta-rz-common/recipes-debian/buster/sources/glibc.inc
index 28893d7d..ef6920e0 100644
--- a/meta-rz-common/recipes-debian/buster/sources/glibc.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/glibc.inc
@@ -1,19 +1,19 @@
 # This is generated by debian-source.bbclass
-DPV = "2.28-10+deb10u3"
+DPV = "2.28-10+deb10u4"
 DPV_EPOCH = ""
 REPACK_PV = "2.28"
 PV = "2.28"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/glibc/glibc_2.28-10+deb10u3.dsc;name=glibc_2.28-10+deb10u3.dsc \
+    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/glibc/glibc_2.28-10+deb10u4.dsc;name=glibc_2.28-10+deb10u4.dsc \
     ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/glibc/glibc_2.28.orig.tar.xz;name=glibc_2.28.orig.tar.xz \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/glibc/glibc_2.28-10+deb10u3.debian.tar.xz;name=glibc_2.28-10+deb10u3.debian.tar.xz \
+    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/glibc/glibc_2.28-10+deb10u4.debian.tar.xz;name=glibc_2.28-10+deb10u4.debian.tar.xz \
 "
 
-SRC_URI[glibc_2.28-10+deb10u3.dsc.md5sum] = "3cf31cd78b313daaf1f2729369e20096"
+SRC_URI[glibc_2.28-10+deb10u4.dsc.md5sum] = "b39dd7f7f54c6b7da3eefcd391470af2"
 SRC_URI[glibc_2.28.orig.tar.xz.md5sum] = "2d78d5b080fbe4fefa2e1ccef9c39dbc"
-SRC_URI[glibc_2.28-10+deb10u3.debian.tar.xz.md5sum] = "b3713583a27b26a1a6e051c3f9447782"
+SRC_URI[glibc_2.28-10+deb10u4.debian.tar.xz.md5sum] = "8ebb6ee80ba6a5e39eacc31c95994ef0"
 
-SRC_URI[glibc_2.28-10+deb10u3.dsc.sha256sum] = "731d162af297ab2f042e73b0910388a84214e48b68766ded409e6c391ca5e9c4"
+SRC_URI[glibc_2.28-10+deb10u4.dsc.sha256sum] = "55e4ebd9a55755c84d42709b23f9b269f46b6a76f5040a0e05cfd323ba67f8af"
 SRC_URI[glibc_2.28.orig.tar.xz.sha256sum] = "53d3c1c7bff0fb25d4c7874bf13435dc44a71fd7dd5ffc9bfdcb513cdfc36854"
-SRC_URI[glibc_2.28-10+deb10u3.debian.tar.xz.sha256sum] = "552ddba370dfe93ae9360b17ad4772b9f72b43223bf5e40a651797a2cbebadc2"
+SRC_URI[glibc_2.28-10+deb10u4.debian.tar.xz.sha256sum] = "dc287870d4b8cb5d1d175fa9b95e3a97d6b68699680b443ae7b2a1b89f0fe8fc"
-- 
2.25.1

